cmd/test-resources/repos/another-repo/some-ttps/cleanup-tests/stress-tests.yaml

```diff
@@ -0,0 +1,29 @@
+name: cleanup-stress-test
+description: |
+  Cleanup a whole bunch of different ways
+steps:
+  - name: file-step-with-inline-cleanup
```

Comment:
All the YAMLs added to this folder are used for new E2E tests in `cmd/run_test.go` - this one demonstrates that we can use all types of actions in `cleanup:` - previously we could only use `inline:` and `file:`